Controller/Adminhtml/Plugin/Proxy.php

```diff
@@ -41,28 +41,26 @@ class Proxy extends AbstractAction
      * @param JsonFactory $resultJsonFactory
      * @param LoggerInterface $logger
      * @param Backend\MagentoAPI|MagentoAPI $magentoAPI
-     * @param RequestRouter $requestRouter
      */
     public function __construct(
         Context $context,
         DataStore $dataStore,
         MagentoIntegration $integrationContext,
         JsonFactory $resultJsonFactory,
         LoggerInterface $logger,
```

Comment:
ohh, another solution would be extending `cloudflare-plugin-backend` to an empty class. This is also fine. Both are hacks anyway :man_shrugging: